commit dd457606ca4583b4a5e83d4e8956e6f9db61df6d
Author: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date:   Thu Feb 10 11:23:24 2022 -0300

    x86_64: Remove bcopy optimizations
    
    The symbols is not present in current POSIX specification and compiler
    already generates memmove call.
    
    (cherry picked from commit bf92893a14ebc161b08b28acc24fa06ae6be19cb)

diff --git a/sysdeps/x86_64/multiarch/bcopy.S b/sysdeps/x86_64/multiarch/bcopy.S
deleted file mode 100644
index 639f02bde3ac3ed1..0000000000000000
--- a/sysdeps/x86_64/multiarch/bcopy.S
+++ /dev/null
@@ -1,7 +0,0 @@
-#include <sysdep.h>
-
-	.text
-ENTRY(bcopy)
-	xchg	%rdi, %rsi
-	jmp	__libc_memmove	/* Branch to IFUNC memmove.  */
-END(bcopy)
